extension/src/init.js

```diff
@@ -16,13 +16,17 @@ if (envConfig.keepAliveTimeout !== undefined)
   server.keepAliveTimeout = envConfig.keepAliveTimeout
 server.listen(port, async () => {
   const ctpProjectKeys = config.getAllCtpProjectKeys()
-  await pMap(ctpProjectKeys, async (ctpProjectKey) => {
-    const ctpConfig = config.getCtpConfig(ctpProjectKey)
-    if (ctpConfig.ensureResources) {
-      const ctpClient = ctpClientBuilder.get(ctpProjectKey)
-      await ensureResources(ctpClient)
-    }
-  })
+  await pMap(
+    ctpProjectKeys,
+    async (ctpProjectKey) => {
+      const ctpConfig = config.getCtpConfig(ctpProjectKey)
+      if (ctpConfig.ensureResources) {
+        const ctpClient = ctpClientBuilder.get(ctpConfig)
+        await ensureResources(ctpClient)
```

Comment:
I've changed this in multitenancy-extension-IT branch, also envConfig to moduleConfig..
```suggestion
await ensureResources(ctpClient, moduleConfig.apiExtensionBaseUrl)
```

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/541/commits/4d0741f4b2d6113bda662ffdf5d7209d4d4faa27